### ISSUE TITLE
search closed when pressing return instead of searching

### ISSUE THREAD

**Issue by binhex** [WDaan/VueTorrent#259]
This is a very minor annoyance, if you click on search, enter in your criteria and press return, then instead of a search being performed it instead closes the search box. 

I would like to request that the focus is moved from the 'X' close search button to the 'SEARCH' button when pressing return, so that a search is performed.

**Comment by binhex** [comment]
hmm i take it back, it now seems to be working correctly, ok that was odd, closing for now, if i can replicate it i will reopen.

**Comment by binhex** [comment]
OK i got to the bottom of it, to replicate the bug you need to close any existing open connection to VueTorrent, then re-open VueTorrent and perform a search, the first time you perform the search you will notice it returns no results and instantly closes, if you then click on search again and put in the same criteria then you do see the results returned.

**Comment by WDaan** [comment]
Yes, Ive noticed as well...the expected behavior is not consistent... It should definetly be fixed

**Comment by binhex** [comment]
@WDaan thanks!.

**Comment by LukePrior** [comment]
I've noticed similar issues

**Comment by jsmdy** [comment]
it depends were your mouse cursor is when you hit enter, if it's on searching form it will works, other places close the window, as an external clic to close window

**Comment by WDaan** [comment]
Seems to be fixed
